soh/soh/SaveManager.cpp

```diff
@@ -218,6 +218,12 @@ void SaveManager::LoadRandomizerVersion2() {
     std::string gregHintText;
     SaveManager::Instance->LoadData("gregHintText", gregHintText);
     memcpy(gSaveContext.gregHintText, gregHintText.c_str(), gregHintText.length());
+    std::string sheikText;
+    SaveManager::Instance->LoadData("sheikText", sheikText);
+    memcpy(gSaveContext.sheikText, sheikText.c_str(), sheikText.length());
+    std::string sariaText;
+    SaveManager::Instance->LoadData("sariaText", sariaText);
+    memcpy(gSaveContext.sariaText, sariaText.c_str(), sariaText.length());
```

Comment:
There's actually a bug here with other hints we need to fix, we need to copy the termination char over (someone can spot check me on this)
```suggestion
    std::string sheikText;
    SaveManager::Instance->LoadData("sheikText", sheikText);
    memcpy(gSaveContext.sheikText, sheikText.c_str(), sheikText.length() + 1);
    std::string sariaText;
    SaveManager::Instance->LoadData("sariaText", sariaText);
    memcpy(gSaveContext.sariaText, sariaText.c_str(), sariaText.length() + 1);
```